lib/bridge.js

```diff
@@ -237,11 +240,109 @@ Bridge.prototype.run = function(port, config, appServiceInstance) {
         }
         self.opts.controller.onLog(line, false);
     });
+    this.customiseAppservice(this.appService);
     this.appService.listen(port);
     return this.loadDatabases();
 };
 
 /**
+ * Apply any customisations required on the appService object.
+ * Subclasses are intended to override this method where appropriate. If
+ * overridden the subclass method will still have to call the one in this
+ * class.
```

Comment:
I primarily split the code out to a new method just to keep it neater contained, away from the man `run` method. There's no pressing concern to document it and make it visible to subclasses. If that makes it awkward, maybe best just to not bother and keep it for internal purposes.